tests/integration/kubernetes/k8s-custom-dns.bats

```diff
@@ -9,6 +9,8 @@ load "${BATS_TEST_DIRNAME}/../../common.bash"
 load "${BATS_TEST_DIRNAME}/tests_common.sh"
 
 setup() {
+	[ "${KATA_HYPERVISOR}" = "qemu-tdx" ] && skip "See: https://github.com/kata-containers/kata-containers/issues/9663"
```

Comment:
Just to check - as it's been failing for a while - I assume that we don't think it's related to shared_fs?

Reply:
Yes, I don't think at all that this is related to shared_fs=none.

Reply:
I was going to ask to split this out into another PR but then you might have troubles to pass the tests on this PR. So yeah, keep it here.

Reply:
IK the SEV tests have gotten stuck on this a lot as well. You might do @AdithyaKrishnan a favor by skipping those here as well, but that can happen separately if you feel it's not in scope.